tests/format/x/html/html-void-tags/jsfmt.spec.js

```diff
@@ -1,3 +1,4 @@
-run_spec(__dirname, ["html", "angular", "vue"]);
+// using bogus option to keep snapshot more consistent with prettierX 0.18.x
+run_spec(__dirname, ["html", "angular"], { bogus: null });
 run_spec(__dirname, ["html"], { htmlVoidTags: true });
-run_spec(__dirname, ["angular", "vue"], { htmlVoidTags: true });
+run_spec(__dirname, ["angular"], { htmlVoidTags: true });
```

Comment:
- [ ] try keeping with `vue`, with added 'errors' options to reproduce the errors encountered with updates from Prettier 2.3.2
- [ ] investigate whether or not this should be considered a regression from prettierX 0.18.x